Services/BookingService.cs

```diff
@@ -0,0 +1,160 @@
+﻿using AirportTicketBooking.Classes;
+using AirportTicketBooking.Repositories;
+using ClassType = TicketClassType.ClassType;
+
+namespace AirportTicketBooking.Routers
+{
+    internal class BookingService
+    {
+        private BookingRepository _bookingRepositoty;
+        private FlightService _flightRouter;
+
+        public BookingService()
+        {
+            _bookingRepositoty = new BookingRepository();
+        }
+
+        private List<Booking> GetAllData()
+        {
+            return _bookingRepositoty.Load();
+        }
+
+        public IEnumerable<Booking> GetAll()
+        {
+            return GetAllData().Where(booking => booking.ValidationMessage == null);
+        }
+
+        public bool TryGet(int bookingId, out Booking booking)
+        {
+            booking = GetAll().FirstOrDefault(booking => booking.BookingId == bookingId);
+            return (booking != null);
+        }
+
+        public void Add(Booking booking)
+        {
+            _bookingRepositoty.Save(booking);
+        }
+
+        public void RemoveBy(int bookingId)
+        {
+            if (!TryGet(bookingId, out _))
+            {
+                throw new Exception($"Booking with BookingId: {bookingId} doesn't exist");
+            }
+
+            var remainingBookings = GetAll().Where(booking => booking.BookingId != bookingId).ToList();
+
+            _bookingRepositoty.DeleteTheFileData();
+
+            foreach (var booking in remainingBookings)
+            {
+                _bookingRepositoty.Save(booking);
+            }
+        }
+
+        public List<Booking> FilterBy(ref List<string> errorList, string passengerPassportId = null, string price = null,
```

Comment:
For your interest, the filtration should be done at the Repo level when we deal with a database so we can load only filtered data. But here since we deal with a file a data storage, then it's fine to do it here since we will already load the data and then filter them.